bin/cml-runner.js

```diff
@@ -405,7 +405,7 @@ const opts = yargs
   .default('idle-timeout', RUNNER_IDLE_TIMEOUT)
   .describe(
     'idle-timeout',
-    'Time in seconds for the runner to be waiting for jobs before shutting down. Setting it to 0 disables automatic shutdown'
+    'Seconds to wait for jobs before shutting down. Set to -1 to disable timeout'
```

Comment:
I would say ```0 or negative number.```

Reply:
While it's what we really do, I think that using -1 is just fine. I'm inclined to think that users don't neccessarily need to know all the possible values, but just the most intuitive one. If they feel rebel, they can always use `0` or `-42` to achieve the same result. 😄

Reply:
> While your suggestion reflects truthfully the program behavior

I agree 

I do not have a strong opinion while it does not introduce a breaking change

Reply:
> _I do not have a strong opinion while it does not introduce a breaking change_

It's backwards-compatible, but I don't have a strong opinion either. 🤷🏼‍♂️  If your mathematical mind can't bear the simplification, I'm fine with your suggestion. 😉

Reply:
FYI on the subject of maths, since zero is a negative number you could really say
```suggestion
    'Seconds to wait for jobs before shutting down. Set to a negative number to disable timeout'
```